src/NHibernate.Validator/Properties/DefaultValidatorMessages.fr.resx

```diff
@@ -135,6 +135,9 @@
   <data name="validator.email" xml:space="preserve">
     <value>Address email mal formée</value>
   </data>
+  <data name="validator.enum" xml:space="preserve">
+    <value>doit correspondre à la définition Enum</value>

```

Comment:
Français syntaxiquement discutable...

Here it should be `doit correspondre à la définition de l'énumération`.